.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: ZipCaptions Discord
+    url: https://discord.gg/Swe2JeHnPc
+    about: If you have questions about using ZipCaptions or need additional help, please join the Discord server.
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)